src/Globe/Globe.js

```diff
@@ -219,7 +221,12 @@ Globe.prototype.getZoomLevel = function getZoomLevel() {
 
 
 Globe.prototype.computeDistanceForZoomLevel = function computeDistanceForZoomLevel(zoom, camera) {
-    return camera.preSSE * Math.pow(this.tiles.minLevel, (this.tiles.maxLevel - zoom + 1)) / SSE_SUBDIVISION_THRESHOLD;
+    const delta = Math.PI / Math.pow(2, zoom - 1);
+    const chord = 2.0 * 6378137 * Math.sin(delta * 0.5);
+    const radius = chord * 0.5;
+    const error = radius / 256;
+    // return camera.preSSE * Math.pow(this.tiles.minLevel, (this.tiles.maxLevel - zoom + 1)) / SSE_SUBDIVISION_THRESHOLD;
+    return camera.preSSE * error / SSE_SUBDIVISION_THRESHOLD;
```

Comment:
Remove dead code.

Also the whole function isn't really clear. What are you computing? What is `chord`? What is `256`? `6378137`?